src/main/java/org/mal_lang/langspec/ttc/TtcDistribution.java

```diff
@@ -16,568 +16,488 @@
+      double rate = arguments[0];
+      requirePositive(rate);
+    }
+
+    @Override
+    public double getMeanTtc(double... arguments) {
+      this.validateArguments(arguments);
+      double rate = arguments[0];
+      return 1.0 / rate;
+    }
+  },
+
+  /**
+   * Enum constant representing a gamma distribution.
+   *
+   * @since 1.0.0
+   */
+  GAMMA("Gamma") {
+    @Override
+    public void validateArguments(double... arguments) {
+      requireFinite(requireArgumentsSize(requireNonNull(arguments), 2));
+      double shape = arguments[0];
+      double scale = arguments[1];
+      requirePositive(shape);
+      requirePositive(scale);
+    }
+
+    @Override
+    public double getMeanTtc(double... arguments) {
+      this.validateArguments(arguments);
+      double shape = arguments[0];
+      double scale = arguments[1];
+      return shape * scale;
+    }
+  },
+
+  /**
+   * Enum constant representing a log-normal distribution.
+   *
+   * @since 1.0.0
+   */
+  LOG_NORMAL("LogNormal") {
+    @Override
+    public void validateArguments(double... arguments) {
+      requireFinite(requireArgumentsSize(requireNonNull(arguments), 2));
+      // double normalMean = arguments[0];
+      double normalStandardDeviation = arguments[1];
+      requirePositive(normalStandardDeviation);
+    }
+
+    @Override
+    public double getMeanTtc(double... arguments) {
+      this.validateArguments(arguments);
+      double normalMean = arguments[0];
+      double normalStandardDeviation = arguments[1];
+      return Math.exp(normalMean + normalStandardDeviation * normalStandardDeviation / 2.0);
+    }
+  },
+
+  /**
+   * Enum constant representing a pareto distribution.
+   *
+   * @since 1.0.0
+   */
+  PARETO("Pareto") {
+    @Override
+    public void validateArguments(double... arguments) {
+      requireFinite(requireArgumentsSize(requireNonNull(arguments), 2));
+      double minimumValue = arguments[0];
+      double shape = arguments[1];
+      requirePositive(minimumValue);
+      requirePositive(shape);
+    }
+
+    @Override
+    public double getMeanTtc(double... arguments) {
+      this.validateArguments(arguments);
+      double minimumValue = arguments[0];
+      double shape = arguments[1];
+      return shape > 1.0 ? (shape * minimumValue) / (shape - 1.0) : Double.MAX_VALUE;
+    }
+  },
+
+  /**
+   * Enum constant representing a truncated normal distribution.
+   *
+   * @since 1.0.0
+   */
+  TRUNCATED_NORMAL("TruncatedNormal") {
+    @Override
+    public void validateArguments(double... arguments) {
+      requireFinite(requireArgumentsSize(requireNonNull(arguments), 2));
+      // double mean = arguments[0];
+      double standardDeviation = arguments[1];
+      requirePositive(standardDeviation);
+    }
+
+    @Override
+    public double getMeanTtc(double... arguments) {
+      this.validateArguments(arguments);
+      double mean = arguments[0];
+      // double standardDeviation = arguments[1];
```

Comment:
could clarify along the lines of "standardDeviation, args[1], not applicable" or similar. At first glance I thought this was just commented code

Reply:
similar to the comment above, `standardDeviation` is not used for determining the mean of a normal distribution